src/segment-loader.js

```diff
@@ -2470,6 +2471,7 @@ export default class SegmentLoader extends videojs.EventTarget {
    * @private
    */
   handleAppendsDone_() {
+    this.trigger('updateend');
```

Comment:
I needed a definitive way to determine when an append is actually done. We have appended but that only fires in a few scenarios. I could have latched on to `updateend`  on source buffers but that was much lower level. Ultimately I think an event fits best, but i'm not sure if the name should stay `updateend`